analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ImplementISerializableCorrectly.cs

```diff
@@ -251,4 +252,46 @@ public virtual String Option
             get { return this.option; }
         }
     }
+
```

Comment:
I would be explicit adding also a compliant example implementing Exception interface without `[Serializable]` attribute.
`public class MyException : Exception`

Reply:
Good idea. I mentioned exception here as well https://github.com/SonarSource/sonar-dotnet/pull/5709#issuecomment-1317011974 I would not do anything special about it for now. In [Best practices for exceptions](https://learn.microsoft.com/en-us/dotnet/standard/exceptions/best-practices-for-exceptions#ensure-that-exception-data-is-available-when-code-executes-remotely) serializability of exceptions is not even mentioned in the "executes remote" section. I don't think a lot of people really care about it, except e.g. akka:
https://github.com/SonarSource/sonar-dotnet/blob/fc6cc877d11c0ccd4267a24c3867b3ad5ed0c63c/analyzers/its/sources/akka.net/src/core/Akka.Streams/StageException.cs#L16-L35